src/service/account-service.js

```diff
@@ -0,0 +1,62 @@
+const uuid = require('uuid');
+const AWS = require('aws-sdk');
+
+const dynamoDB = 
+new AWS.DynamoDB.DocumentClient({
+  region: 'localhost',
+  endpoint: 'http://localhost:8000'
```

Comment:
Should be part of configuration so it can be different per environment. You can address this in a separate PR.

Reply:
Separate PR, todays assigment.